src/css/your-order.css

```diff
@@ -0,0 +1,206 @@
+  font-weight: 400;
+  font-size: 14px;
+  letter-spacing: -0.02em;
+  color: rgba(252, 249, 249, 0.5);
+  background-color: transparent;
+  transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.form-comments {
+  width: 335px;
+  height: 196px;
+  margin-bottom: 40px;
+  border: 1px solid rgba(252, 249, 249, 0.1);
+  border-radius: 8px;
+  padding: 14px 16px;
+  font-weight: 400;
+  font-size: 14px;
+  letter-spacing: -0.02em;
+  color: rgba(252, 249, 249, 0.5);
+  background-color: transparent;
+  transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.form-info:hover,
+.form-info:focus,
+.form-comments:hover,
+.form-comments:focus {
+  border: 1px solid rgba(252, 249, 249, 0.6);
+  color: #fcf9f9;
+}
+
+.btn-order {
+  width: 145px;
+  height: 49px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  border: 1px solid rgba(252, 249, 249, 0.1);
+  border-radius: 15px;
+  padding: 14px 36px;
+  background-color: transparent;
+  font-weight: 600;
+  font-size: 14px;
+  letter-spacing: -0.04em;
+  text-transform: uppercase;
+  color: #fcf9f9;
+}
+
+.form-icon {
+  margin-left: 10px;
+  fill: #ecd979;
+}
+
+.btn-order:hover,
+.btn-order:focus {
+  border: 1px solid #ecd979;
+}
+
+.order-picture {
+  display: none;
+}
+
+/* ======================TABLET=================== */
+
+@media screen and (min-width: 768px) {
+  .backgroung-img {
+    width: 768px;
+    height: 776px;
+    background-image: url(/img/your-order/green-girl-tablet\ 12.21.16.jpg);
+    margin-bottom: 100px;
+  }
+
+  @media screen and (min-resolution: 2dppx), (min-resolution: 192dpi) {
+    .backgroung-img {
+      background-image: url(/img/your-order/green-girl-tablet@2x\ 12.21.16.jpg);
+    }
+  }
+  .order {
+    padding-bottom: 49px;
+    background-image: url(../img/gray-ornament.png);
+    background-size: 545px 484px;
+    background-position: bottom left -255px;
+    background-repeat: no-repeat;
+    padding-bottom: 0;
+  }
+
+  .order-titel {
+    width: 542px;
+    height: 174px;
+    line-height: 1.32;
+  }
+
+  .form-info {
+    width: 508px;
+    height: 50px;
+    font-size: 16px;
+  }
+
+  .form-comments {
+    width: 508px;
+    height: 196px;
+    font-size: 16px;
+  }
+
+  .btn-order {
+    width: 168px;
+    height: 50px;
+    margin-bottom: 100px;
+    margin-left: 0;
+  }
+
+  .order-picture {
+    display: block;
+    border-radius: 4px;
+    margin-bottom: 49px;
+  }
+}
+
+/* =============================DT 1280px================================= */
+
+@media screen and (min-width: 1280px) {
+  .order-titel {
+    width: 542px;
+    padding-bottom: 0;
+  }
+
+  .backgroung-img {
+    width: 100%;
+    height: 776px;
+    padding: 0;
+    background-image: url(/img/your-order/green-girl-d1280\ 12.21.16.jpg);
+    margin-bottom: 100px;
+  }
+
+  @media screen and (min-resolution: 2dppx), (min-resolution: 192dpi) {
+    .backgroung-img {
+      background-image: url(/img/your-order/green-girl-d1280@2x\ 12.21.16.jpg);
+    }
```

Comment:
@media screen and (min-resolution: 2dppx), (min-resolution: 192dpi) {
    .backgroung-img {
        background-image: url(/img/your-order/green-girl-d1280@2x.jpg);
    }